api/swagger.yml

```diff
@@ -645,6 +645,10 @@ components:
         force:
           type: boolean
           default: false
+        allow_no_changes:
```

Comment:
1. I guess we will need to document the flag above, too!  In particular, what happens if I set `force` but not `allow_no_changes`?  In order to keep the minor version guarantee, I think that the merge has to fail.  This will not be clear to users not aware of the history we are now creating.

   So I _insist_ on documenting what force does (sorry, it should have been done before), and I _think_ that it should be documented as not allowing an empty merge.
1. I believe that this flag should be called allow_empty.  It is already called allow_empty for commit and for revert.  I don't want to learn another term, and I don't like boolean flags whose names are negative.